game/templates/game/lesson_detail.html

```
@@ -268,6 +292,34 @@ <h3 id="example-title"></h3>
+        <h2>🎯 Practice Position</h2>
+
+        <p id="lesson-instruction"></p>
+
+        <div class="lesson-board-wrapper">
+            <div id="practice-board" class="lesson-board"></div>
+        </div>
+
+        <p id="lesson-result"></p>
+
+        <button
+            id="retry-btn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)